src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -237,7 +244,7 @@ public async Task<Cohort> CreateCohort()
 
             public void VerifyProviderCohortCreation()
             {
-                Provider.Verify(x => x.CreateCohort(It.Is<AccountLegalEntity>(ale => ale == AccountLegalEntity.Object), It.IsAny<DraftApprenticeshipDetails>()));
+                Provider.Verify(x => x.CreateCohort(It.Is<AccountLegalEntity>(ale => ale == AccountLegalEntity.Object), It.IsAny<DraftApprenticeshipDetails>(), Originator.Provider));
```

Comment:
It's worth verifying this call is made with the same `DraftApprenticeshipDetails` instance. I think the `It.Is<AccountLegalEntity>(ale => ale == AccountLegalEntity.Object)` call could also be simplified to just `AccountLegalEntity.Object`.